codec/legacy/codec.go

```diff
@@ -15,5 +16,16 @@ func init() {
 	Cdc = codec.NewLegacyAmino()
 	cryptocodec.RegisterCrypto(Cdc)
 	codec.RegisterEvidences(Cdc)
-	Cdc.Seal()
+}
+
+// PrivKeyFromBytes unmarshals private key bytes and returns a PrivKey
+func PrivKeyFromBytes(privKeyBytes []byte) (privKey cryptotypes.PrivKey, err error) {
+	err = Cdc.UnmarshalBinaryBare(privKeyBytes, &privKey)
+	return
+}
+
+// PubKeyFromBytes unmarshals public key bytes and returns a PubKey
+func PubKeyFromBytes(pubKeyBytes []byte) (pubKey cryptotypes.PubKey, err error) {
+	err = Cdc.UnmarshalBinaryBare(pubKeyBytes, &pubKey)
+	return
```

Comment:
Why adding this functions? Anyone can use `Cdc.UnmarshalBinaryBare` with any interface registered to `amino`. What's the advantage of adding this functions? If for documentation then I would prefer using tests or a manual (docs).

Reply:
Otherwise we will need to add this small functions to all interfaces.

Reply:
I believe it's not adding, but just moving from `crypto/` into this `legacy/` package.

Reply:
yeah, just moving from `crypto/codec/amino.go`